<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="rotationSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>681</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="scaleSlider">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>681</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>4</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="rotateScale">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>104</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>obracanie</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>631</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string>zdjecie</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioScale">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>104</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>skalowanie</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>otworz zdjecie</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
